ci/scripts/utils/launch_java_agent.sh

```diff
@@ -84,6 +84,10 @@ esac
 LOG=lanuched_agent-$(date +%Y%m%d%M).log
 rm -f "${LOG}"
 
+#shellcheck disable=SC2034
+# Needed for the module-setup.sh script
+# but did not want to set it globally
+HOMEgfs="${HOMEGFS_}"
 source "${HOMEGFS_}/ush/module-setup.sh"
```

Comment:
If this is only needed for `module-setup.sh`, then you could instead do the following.

```suggestion
HOMEgfs="${HOMEGFS_}" source "${HOMEGFS_}/ush/module-setup.sh"
```

Reply:
Yes I was thinking of doing it that way too.